### PR TITLE
fix bugs on _convert_layers_batchnorm when scale is false

### PR DESCRIPTION
**Pull request by chenghaz**

### CODE REVIEW

**Comment by namizzz** [comment]
Hi @chenghaz ,Which version Tensorflow are you using?

**Comment by chenghaz** [comment]
@namizzz My tensorflow version is 1.9.0

**Comment by chenghaz** [comment]
@namizzz  Also, I found that if I use tf.layers.batch_normalization, the scope[-3] at "tensorflow_frozenparser.py" line 621, is 'batch_normalization' instead of 'BatchNorm'

**Comment by namizzz** [comment]
Hi @chenghaz ,ok. Maybe people using other version of TF can't use MMdnn if I merge your PR. Could you please provide a general PR for different version (1.7  1.8  1.9) using "try expect" ? Thanks!

**Comment by chenghaz** [comment]
Please check it again. I double check tensorflow version 1.8 and 1.9. It worked on Inception v1 - v4, resnet and mobilenet.

**Comment by kitstar** [comment]
Hi @chenghaz  merged. Thanks for the contribution!
